jsonnet/prometheus.jsonnet

```diff
@@ -156,6 +159,29 @@ local namespacesRole = policyRule.new() +
       secret.mixin.metadata.withNamespace($._config.namespace) +
       secret.mixin.metadata.withLabels({ 'k8s-app': 'prometheus-k8s' }),
 
+    kubeRbacProxySecret:
+      local config = {
+        'resource-attributes.yaml': std.base64(std.manifestYamlDoc({
```

Comment:
as this now contains more than resource attribute configuration, let's rename this to `config.yaml`